pom_confluent.xml

```diff
@@ -593,6 +594,26 @@
         </dependency>
 
         <!--junit for unit test-->
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>${junit-jupiter.version}</version>
```

Comment:
Can we use [junit bom](https://mvnrepository.com/artifact/org.junit/junit-bom/5.10.2) instead of specifying a version for each dependency?